server/routes/appointments/create-and-edit/appointmentJourney.ts

```diff
@@ -19,13 +20,7 @@ export type AppointmentJourney = {
   type: AppointmentType
   createJourneyComplete?: boolean
   appointmentName?: string
-  prisoners?: {
-    number: string
-    name: string
-    prisonCode: string
-    status: string
```

Comment:
Re-factored to [here](https://github.com/ministryofjustice/hmpps-activities-management/pull/757/files#diff-9b2f9e6729e74ffce4fd2a91df6a77b758cde996e6659ca2293bd54c0189c0e9)